x/epochs/spec/README.md

```diff
@@ -27,12 +27,16 @@ keeping own code for epochs.
 
 ## State
 
-Epochs module keeps `EpochInfo` objects and modify the information as
+Epochs module keeps [`EpochInfo`]({TODO: Link to proto}) objects and modify the information as
 epochs info changes. Epochs are initialized as part of genesis
-initialization, and modified on begin blockers or end blockers.
+initialization, and modified on begin blockers.
 
 ### Epoch information type
 
+What do folks think about deleting this subsection entirely? Or how do we re-adjust it to be useful?
+I think we need to communicate that epochs are have unique identifiers, and conditions around when
+epoch numbers first start.
+
 ```protobuf
 message EpochInfo {
     string identifier = 1;
```

Comment:
What do folks think, is deleting this proto from the spec helpful?

I'd rather refer people to one well documented proto struct / its relevant go code. I think realistically those code comments are going to be the most helpful for discoverability.

Reply:
agreed on deleting this subsection